coral/src/app/pages/not-found/index.tsx

```diff
@@ -8,13 +8,13 @@ const NotFound = () => {
           Page not found
         </Typography.Heading>
 
-        <Typography.LargeText>
+        <Typography.DefaultStrong>
```

Comment:
I think the recommended substitution is `LargeStrong`

![Screenshot 2023-10-09 at 10 20 16](https://github.com/Aiven-Open/klaw/assets/20607294/1bf78e72-f971-4cad-b5fa-9bd81aa1d303)